changelogs/unreleased/gh-8576-set-compat-options-to-new.md

```diff
@@ -0,0 +1,11 @@
+## feature/compat
```

Comment:
>     NO_DOC=tarantool.io compat doc doesn't state current defaults

It is not clear when defaults in online documentation should be updated.

Should we add a doc request in commit message?

Reply:
As I've stated in `NO_DOC`, there is nothing about the current defaults in the [documentation](https://www.tarantool.io/en/doc/latest/reference/reference_lua/compat/).